code/src/layouts/tables/orders-table/components/orders-table-body/OrdersTableBody.tsx

```diff
@@ -0,0 +1,50 @@
+import { orderBadgeVariants } from "@/layouts/order-item/OrderItem.constants";
+
+import AppBadge from "@/components/app-badge/AppBadge";
+import { AppTableCell } from "@/components/app-table/components";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import { orderStatuses } from "@/constants/orderStatuses";
+import { Order } from "@/types/order.types";
+import formatDate from "@/utils/format-date/formatDate";
+import formatPrice from "@/utils/format-price/formatPrice";
+
+import "@/layouts/tables/orders-table/components/orders-table-body/OrdersTableBody.scss";
+
+type OrderTableBodyProps = {
+  order: Order;
+};
+
+const OrdersTableBody = ({ order }: OrderTableBodyProps) => {
+  const orderItemStatus = orderStatuses[order.orderStatus];
+  const orderReciever = `${order.receiver.firstName} ${order.receiver.lastName}`;
+
+  const ordersTotalPrice = order.orderItems.reduce(
+    (total, item) => total + item.quantity * item.product.price,
+    0
+  );
```

Comment:
keep in mind. Maybe `totalPrice` will be handled from backend side

Reply:
I created the issue and already assigned for refactoring order type. It also includes to remove this reduce functions. But for now it works in this way